apps/resize/resize.cpp

```diff
@@ -152,7 +152,7 @@ int main(int argc, char **argv) {
         Halide::Runtime::Buffer<>::make_interleaved(in.type(), in.width(), in.height(), in.channels());
     auto out_packed =
         Halide::Runtime::Buffer<>::make_interleaved(out.type(), out.width(), out.height(), out.channels());
-    time = Halide::Tools::benchmark(10, 10, [&]() { resize_fn(in_packed, scale_factor, out_packed); });
+    time = Halide::Tools::benchmark([&]() { resize_fn(in_packed, scale_factor, out_packed); });
```

Comment:
Meta-comment: I have a private branch in which  I can run benchmarks with this or GTest-bench easily; I'll compare some problematic cases between this and that (and my original proposal) and see how the results compare.